### ISSUE TITLE
Update 1.5.68 causes FEN not to work correct

### ISSUE THREAD

**Issue by ANHBOB** [Tikipeter/repository.tikipeter#54]
After update Nov. 13  after scraping and choosing Trakt watchlist movie a box pops up giving multiple options. Never saw that before the update...usually the movie just starts to play.

This morning Nov. 14 I go to Trakt Collections TV shows to watch one I missed last night Chicago PD..Season 7...Nothing is there...no episodes.  Opened The Crew addon...it worked with no problems.

**Comment by Tikipeter** [comment]
What does the box popping up giving multiple options say? What are the multiple options? Is it after playback has started, or before? There were changes to the subtitle implementation in the last update, so I assume it could be that, but you don't give enough info to be sure.

If it is the subtitles, then go to Settings->Playback->Subtitles and set 'Downloads Subtitle Action' to 'Auto' or 'Off'.

The tv show problem is due to TVDb API v2 malfunctioning and not returning correct info. Exodus forks use v1 of the API, so are unaffected. Nothing to do with the last update, just a bad coincidence. Any show with over 100 episodes will be affected.

I will push an update soon (tikimeta) to hopefully fix this issue, but you will need to clear the Meta Cache for the changes to take effect.

**Comment by ANHBOB** [comment]
The box popped up after 1) scrapped for a movie 2) chose the scraper...between the scraper slection and the movie starting to play is when the box popped up...it said  1ENG 1080 (scraper),2ENG 1080 (scraper) all the way through 9ENG 1080 (scraper). It didn't say anything about subtitles. 

I saw you pushed out an update...I updated FEN..then I cleared the Meta Cache....everything is working beautifully once again.

I really appreciate all the work you've done with FEN.....I'm sure others do as well. It's my number 1 go to addon above all others.  I've never used one that works as good as yours does. Keep up the great work.  I only have 2 addons installed...FEN and The Crew.  Its funny but usually when The Crew has an update FEN has an update at the same time.
